src/pages/patientView/PatientViewPage.tsx

```diff
@@ -307,21 +307,24 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                             />
 
                             {
-                                (patientViewPageStore.mutationData.isComplete && patientViewPageStore.cnaSegments.isComplete && sampleManager) && (
-                                    <GenomicOverview
-                                        mergedMutations={patientViewPageStore.mergedMutationData}
-                                        cnaSegments={patientViewPageStore.cnaSegments.result}
-                                        sampleOrder={sampleManager.sampleIndex}
-                                        sampleLabels={sampleManager.sampleLabels}
-                                        sampleColors={sampleManager.sampleColors}
-                                        sampleManager={sampleManager}
-                                        getContainerWidth={()=>$(window).width()}
-                                    />
+                                (patientViewPageStore.mutationData.isComplete && patientViewPageStore.cnaSegments.isComplete && sampleManager)
```

Comment:
I think test for this logic really belongs in end-to-end.  Just too difficult to mock this page.